app/src/main/res/raw/aaaadb.json

```diff
@@ -260,6 +355,16 @@
       "displaytext": "Regarder les nouvelles",
       "transcript": "On va regarder les nouvelles courantes, euh oui qu'est-ce qu'il s'y passe, euh, voilà la presse nationale, on va regarder peut-être les informations"
     },
+    {
+      "name": "roland_kunings",
+      "displaytext": "Roland kunings",
+      "transcript": "Roland kunings"
```

Comment:
Hahaha Keunings pas Kunings
https://perso.uclouvain.be/roland.keunings/

Reply:
Oh non, pardon Roland 😢